src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -331,7 +385,60 @@
   end
 
   describe '#setup_codemirror_editor' do
-    skip('Please add some tests')
+    it 'sets the style for the editor' do
+      allow_any_instance_of(Webui::WebuiHelper).to receive(:content_for).with(:content_for_head, kind_of(String))
+      expect_any_instance_of(Webui::WebuiHelper).to receive(:content_for).with(:head_style, kind_of(String))
+      setup_codemirror_editor
+    end
+
+    it 'increases the uid' do
+      allow_any_instance_of(Webui::WebuiHelper).to receive(:content_for)
+      expect(setup_codemirror_editor).to eq(0)
+      expect(setup_codemirror_editor).to eq(1)
```

Comment:
How this is supposed to work?
In line 396 it should equal 0 and in line 397 it should equal 1?